src/ConnectionManager.ts

```diff
@@ -249,6 +249,15 @@ export class ConnectionManager extends EventEmitter {
         return this.connections.find((c) => c.connectionId === connectionId && c.roomId === roomId);
     }
 
+    public validateCommandPrefix(roomId: string, config: IConnectionState, currentConnection?: IConnection) {
+        if (config.commandPrefix === undefined) return;
+        for (const c of this.getAllConnectionsForRoom(roomId)) {
+            if (c != currentConnection && c.conflictsWithCommandPrefix(config.commandPrefix)) {
+                throw new ApiError(`Command prefix "${config.commandPrefix}" is already used in this room. Please choose another prefix.`);
```

Comment:
Should this ApiError return something like a conflict error rather than the default unknown error code?

Reply:
Yes it should; I had missed the status code property of ApiError.